src/screens/BookList/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
+
+import { getBooks } from 'services/BookService';
+import { IBook } from 'types/book.interface';
+
+import styles from './styles.module.scss';
+
+function BookList() {
+  const [data, setData] = useState<IBook[]>([]);
+
+  useEffect(() => {
+    const fetchBooks = async () => {
```

Comment:
this is okay, but you aren't using the react query library, please complete the challenge with it.